test/local/routes/account_devices.js

```diff
@@ -183,9 +183,13 @@ describe('/account/device', function () {
 describe('/account/devices/notify', function () {
   var config = {}
   var uid = uuid.v4('binary')
+  var deviceId = crypto.randomBytes(16)
+  var mockLog = mocks.spyLog()
   var mockRequest = mocks.mockRequest({
+    log: mockLog,
     credentials: {
-      uid: uid.toString('hex')
+      uid: uid,
+      deviceId: deviceId
```

Comment:
The test was wrong here, the `request.credentials` object contains uids in raw bytes, not hex strings.